### PR TITLE
Fix connect long URL styling

### PR DESCRIPTION
**Pull request by rekmarks**
Long URLs were overflowing on the connected sites page.

<details>
<summary>Before</summary>

</details>

<details>
<summary>After</summary>

</details>

### CODE REVIEW

**Review by brad-decker — DISMISSED** [review]

File: ui/app/components/app/permissions-connect-header/index.scss [hunk 1]
```diff
@@ -38,7 +38,14 @@
     color: $Grey-500;
   }
 
+  &__text {
+    width: 100%;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    direction: rtl;
```
Comment (Gudahtt) [inline ui/app/components/app/permissions-connect-header/index.scss:45]: Hah, I was about to comment on this! You beat me to it.

Comment (rekmarks) [reply ui/app/components/app/permissions-connect-header/index.scss:45]: Haha, I was like, "DAMN IT DAMN IT DAMN IT PUSH PUSH PUSH" 😂

File: ui/app/components/app/permissions-connect-header/index.scss [hunk 2]
```diff
@@ -38,7 +38,14 @@
     color: $Grey-500;
   }
 
+  &__text {
+    width: 100%;
```
Comment (Gudahtt) [inline ui/app/components/app/permissions-connect-header/index.scss:42]: The direction here will be incorrectly reversed on right-to-left domains - we can stop this from happening with a `rtl:ignore` directive:
```suggestion
    /*rtl:ignore*/
    width: 100%;
```

Comment (rekmarks) [reply ui/app/components/app/permissions-connect-header/index.scss:42]: TIL! 05f088c

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**